hawc/apps/assessment/forms.py

```diff
@@ -127,9 +128,8 @@ def helper(self):
 
 
 class AssessmentDetailForm(forms.ModelForm):
-    CREATE_LEGEND = "Add additional Assessment details"
-    CREATE_HELP_TEXT = ""
-    UPDATE_HELP_TEXT = "Update additional details for this Assessment."
+    LEGEND = "Additional Assessment Details"
```

Comment:
I ended up adding the same here as well; the text is generic enough it should work for creates and updates